algebird-core/src/main/scala/com/twitter/algebird/statistics/GaussianDistributionMonoid.scala

```diff
@@ -0,0 +1,28 @@
+package com.twitter.algebird.statistics
+
+import com.twitter.algebird.Monoid
+
+/**
+ * @mean - Mean
+ * @sigma - Standard deviation
+ * aka Normal distribution
+ */
+case class GaussianDistribution(mean: Double, sigma: Double)
```

Comment:
Or possibly take it further and integrate with https://github.com/jliszka/probability-monad